runtime/vendored/asserts.bundle.js

```diff
@@ -2,786 +2,912 @@
 // deno-lint-ignore-file
 // This code was bundled using `deno bundle` and it's not recommended to edit it manually
 
-const { Deno  } = globalThis;
+const { Deno } = globalThis;
```

Comment:
We don't want to (re)format vendored files. Could you please update [.prettierignore](https://github.com/filecoin-station/zinnia/blob/main/.prettierignore) to skip everything in `runtime/vendored`?

Either in this PR, or else:
1. Revert changes in this file, land the rest of this PR.
2. Open a new PR to fix prettier ignore config.

Reply:
> I ran prettier, was it right like this?
> `npx prettier -w .`

Yes, that looks good to me!